files/ja/web/api/mediaquerylist/change_event/index.html

```diff
@@ -11,32 +11,49 @@
   - onchange
   - イベントハンドラー
   - プロパティ
-  - メディアクエリ
-translation_of: Web/API/MediaQueryList/onchange
-original_slug: Web/API/MediaQueryList/onchange
+  - メディアクエリー
+translation_of: Web/API/MediaQueryList/change_event
+original_slug: Web/API/MediaQueryList/change_event
 ---
 <p>{{APIRef("CSSOM")}}</p>
-
-<p><code><strong>onchange</strong></code> は {{DOMxRef("MediaQueryList")}} インターフェイスのプロパティで、 {{domxref("MediaQueryList/change_event", "change")}} イベントが発行されたとき、すなわちメディアクエリの対応の状態が変化したときに呼び出される関数を表します。イベントオブジェクトは {{DOMxRef("MediaQueryListEvent")}} のインスタンスであり、古いブラウザーからは後方互換性のために <code>MediaListQuery</code> のインスタンスと解釈されます。</p>
+<p>{{DOMxRef("MediaQueryList")}} インターフェイスの <code><strong>change</strong></code> イベントは、メディアクエリーの状態が変化した時に発生します。</p>
 
 <h2 id="Syntax" name="Syntax">構文</h2>
 
-<pre class="syntaxbox notranslate"><var>MediaQueryList</var>.onchange = function() { ... };</pre>
+<p>{{domxref("EventTarget.addEventListener", "addEventListener()")}} 等のメソッドでイベント名を利用するか、イベントハンドラプロパティを設定します。</p>
```

Comment:
細かいところですが、「イベントハンドラープロパティ」と長音を使用してください。

```suggestion
<p>{{domxref("EventTarget.addEventListener", "addEventListener()")}} 等のメソッドでイベント名を利用するか、イベントハンドラープロパティを設定します。</p>
```